packages/netlify-cms-ui-default/src/AuthenticationPage.js

```diff
@@ -13,8 +13,10 @@ const StyledAuthenticationPage = styled.section`
 `;
 
 const CustomIconWrapper = styled.span`
-  width: 300px;
-  height: 200px;
+  width: auto;
+  height: auto;
+  max-width: 300px;
+  mac-height: 200px;
```

Comment:
@adamcnaglich do you mean `max-height` ?

Reply:
Dear God